lldb/packages/Python/lldbsuite/test/make/Makefile.rules

```diff
@@ -103,6 +103,17 @@ else
 endif
 endif
 
+#----------------------------------------------------------------------
```

Comment:
Thank you for figuring out the changes we need in this file. I tried to do that last week but I failed and I know see what I was missing. <3